core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -451,6 +542,58 @@ private ExprValue exprDate(ExprValue exprValue) {
     }
   }
 
+  /**
+   * DateTime implementation for ExprValue.
+   *
+   * @param dateTime ExprValue of String type.
+   * @param timeZone ExprValue of String type.
+   * @return ExprValue of date type.
+   */
+  private ExprValue exprDateTime(ExprValue dateTime, ExprValue timeZone) {
+    String defaultTimeZone = TimeZone.getDefault().getID();
+    DateTimeFormatter formatDT = DateTimeFormatter.ofPattern("uuuu-MM-dd HH:mm:ss[xxx]")
+        .withResolverStyle(ResolverStyle.STRICT);
+
+    try {
+      LocalDateTime ldtFormatted = LocalDateTime.parse(dateTime.stringValue(), formatDT);
```

Comment:
You can use `new ExprDatetimeValue(dateTime.stringValue())` and don't worry about parser and format.
In that case you just need to update `ExprDatetimeValue` ctor to use `ResolverStyle.STRICT` - it will be good in any case.

Reply:
I don't understand what the benefit of this change would be.

Reply:
You can reduce a bit your code an delegate parsing to code that already exists and covered by tests.

Reply:
Resolved.